hooks/pre_build

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Register qemu-*-static for all supported processors except the 
+# current one, but also remove all registered binfmt_misc before
+docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
Why do we have to run this image? Is it related to the copying of binaries from QEMU into final image?